docs/labs/4-ACM.md

```diff
@@ -350,6 +361,17 @@ you can call ```getWidth()``` or ```getHeight()```.
 You can use this in combination with a ```setLocation(x, y)```
 that will change the object's location if you want the object to be placed in a particular area.
 
+```java
+GImage image = new GImage("media/photo.png", START_X, START_Y);
+image.setSize(400, 400);
+add(image);
+```
+
+Just like the other ACM classes we've introduced so far, ```GImage``` has a nicely compiled summary
+page [here](https://cs.stanford.edu/people/eroberts/jtf/javadoc/student/acm/graphics/GImage.html)
+that shows all of its methods and different uses. Documentation like this is handy for reference
+if you happen to forget anything specific.
```

Comment:
Again split into sentence/clause structure, but perhaps if you linked to it above, we need maybe only the last sentence and another sentence just before to remind them to look at the documentation?